pagopa/pn-frontend
# feat/PN-2993 Disservizi in dettaglio notifica

## Short description
Add info about relevant downtimes in the notification detail page, in pa-webapp, pf-webapp and pg-webapp.

## List of changes proposed in this pull request
- New component `NotificationRelatedDowntimes`.
- Added `downtimeEvents` and `downtimeLegalFactUrl` to the Redux notification slice, and the corresponding actions.
- Include `NotificationRelatedDowntimes` in `NotificationDetail.page` in the three apps, passing the actions and selected info from the Redux store.

## How to test
Open a notification detail, a new frame including downtime info should appear, if there are downtimes relevant to the selected notification. In principle, the relevant downtimes are those occurring between Accepted and EffectiveDate / Viewed events. Some special cases apply, e.g. cancelled notifications shouldn't show any downtimes, for "open" notifications the relevant time frame ends in present time.
